test-data/unit/check-super.test

```diff
@@ -280,8 +280,12 @@ class B(A):
 
 
 [case testSuperOutsideMethodNoCrash]
-class C:
-    a = super().whatever  # E: super() outside of a method is not supported
+class A:
+    x = 1
+class B(A): pass
+class C(B):
+    b = super(B, B).x
+    a = super().whatever  # E: "super" outside of a method is not supported
```

Comment:
The wording with parentheses might be better here because it's just super *without* arguments that doesn't work outside a method.